LLama/Native/NativeApi.cs

```diff
@@ -25,6 +26,9 @@ public unsafe partial class NativeApi
     {
         static NativeApi()
         {
+            // Try to load a preferred library, based on CPU feature detection
+            TryLoadLibrary();
```

Comment:
The return value is ignored here, does the library loading still work?

Reply:
That should be fine.

The `DllImport` methods still work as normal. So if this fails to load anything they will just try to load the DLL as normal when called.

This is actually really important because on MacOS and Linux the `TryLoadLibrary` method _does nothing_! It always "fails" and falls back to the normal behaviour.